scripts/sandbox.ts

```diff
@@ -310,10 +310,18 @@ async function updateStoriesField(mainConfig: ConfigFile, isJs: boolean) {
 
   // If the project is a JS project, let's make sure any linked in TS stories from the
   // renderer inside src|stories are simply ignored.
-  const updatedStories = isJs
+  let updatedStories = isJs
     ? stories.map((specifier) => specifier.replace('js|jsx|ts|tsx', 'js|jsx'))
     : stories;
 
+  if (process.env.CI) {
+    logger.log('Skipping error stories to avoid failures in CI...');
+    // TODO: once we have story tags in place, we can filter error tags instead of doing this
+    updatedStories = updatedStories.map((specifier) =>
+      specifier.replace('src/**/*.stories.@', 'src/**/!(errors)*.stories.@')
+    );
+  }
```

Comment:
@shilman don't all our tools have parameters that allow us to skip certain stories?

Reply:
@yannbf